docs/articles/opinion/importance_of_governance.md

```diff
@@ -0,0 +1,55 @@
+
+The Internet has enabled collaboration from individuals residing in
+different countries and jurisdictions. These jurisdictions are not always compatible with each other, which leads to all sorts of issues
+that affect productivity between them. For example, someone in Venezuela can't easily send a payment to another party in the US without incurring extremely expensive foreign exchange fees. Or think about someone in Cuba trying to open a business with a Costa Rican partner, each party would need to open their own subsidiary in each country, and in the case of Cuba, the best you can hope for is a 50-50 joint venture with the government [1]:  taxes and laws are so different that having a common company is impossible.
+
+So, to build unstoppable organizations, you need to start by laying some ground rules that everyone participating agrees with. More importantly, you need to be able to *adapt* your governance systems to a constantly changing environment, and do it in a way transparent enough so that all participants can clearly see why any decision was made.
+
+In a Medium article by Fred Ehrsam [2] he says that governance is the most vital problem in the space, and fundamental problems can be solved by creating the right incentives for people with adequate skills to solve the problem. A project is just as good as the people behind it. If you have a clear governance process, you can pull talent from wherever it resides, and avoid painful arguments that leave nothing more than people screaming at each other over a mailing list.
+
+There are multiple projects besides Aragon trying to define new ways of Governance that can endure. [Decred](https://decred.org/), for example,
+aims to solve the problems that Bitcoin has in its governance by implementing a consensus voting model that empowers stakeholders
+and allows for the seamless transition from one set of rules to another. This makes the project move at a faster pace
+than Bitcoin since all stakeholders incentives are aligned with the development of the project itself.
+Governance issues have plagued Bitcoin with multiple forks, community splits, and inability to move forward
+on certain [Bitcoin Improvement Proposals](https://github.com/bitcoin/bips).
+
+Aragon takes governance one step further, by not just allowing voting, but providing a platform to implement whole digital jurisdictions.
+The value of this can't be underestimated, a digital jurisdiction enables all participants to know the rules and automatically (via smart
+contracts) incentivizes good behavior by rewarding it, while punishing lechers or people trying to harm the organization. It also
+completely erases the problem with incompatible jurisdictions, since you are now playing by the rules set by a group of people that have
+similar goals, and we all know 2 heads (or a million) are better than 1!
+
+[1] http://www.businessinsider.com/r-cuba-inc-open-to-foreign-capitalists-but-within-limits-2015-6?international=true&r=US&IR=T
```

Comment:
The combined use of links and footnotes throughout the article results in inconsistent styling. Use either/or but not both.